saleor/graphql/decorators.py

```diff
@@ -72,3 +90,21 @@ def check_perms(context):
 staff_member_required = account_passes_test(
     lambda context: context.user.is_active and context.user.is_staff
 )
+
+
+def check_attribute_required_permissions():
+    def check_perms(context, root):
```

Comment:
Maybe we should rename `root` to `attribute` here and to `attribute_value` in the other decorator? `root` is obvious in resolvers, here it may be harder to tell what it represents at a first glance.

Reply:
Another solution would be to add typings.